main.go

```diff
@@ -3,16 +3,15 @@ package main
 import (
 	"github.com/alechewitt/code-wall/api"
 	"github.com/gorilla/mux"
-	"mime"
 	"net/http"
 	"os"
-	"path/filepath"
 )
 
 func main() {
 	r := mux.NewRouter()
 	r.HandleFunc("/", indexHandler)
-	r.HandleFunc("/dist/{folder}/{filename}", fileServer)
+	r.PathPrefix("/dist/").Handler(staticHandler("/dist/", "./dist/"))
+	r.PathPrefix("/node_modules/").Handler(staticHandler("/node_modules/", "./node_modules/"))
```

Comment:
Nice cleanup of the file server! 
One thing, Is there any reason you were serving up the whole node_modules directory? I don't think we should serve out all the modules.. probably better to create a gulp task to copy the libs you need into the dist folder. Or use a cdn?